src/test/java/uk/gov/digital/ho/pttg/application/namematching/candidates/NamesWithAbbreviationSpaceCombinationsAbbreviatedNamesFunctionsTest.java

```diff
@@ -0,0 +1,147 @@
+package uk.gov.digital.ho.pttg.application.namematching.candidates;
+
+import org.junit.Test;
+import uk.gov.digital.ho.pttg.application.namematching.InputNames;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.digital.ho.pttg.application.namematching.candidates.AbbreviatedNamesFunctions.doesNotContainAbbreviatedNames;
+
+public class NamesWithAbbreviationSpaceCombinationsAbbreviatedNamesFunctionsTest {
```

Comment:
I think the names of the test classes have gone a bit awry. Again, this may have been before you started on this task - it's been a little while since I properly looked at this stuff.

Normally if the class is called SomeClass, the unit tests live in SomeClassTest. Sometimes when there's a lot of tests we have multiple called SomeClass[SomeFurtherInfo]Test. But I don't think there's a class called NamesWithAbbreviationSpaceCombinationsAbbreviatedNames or even NamesWithAbbreviationSpaceCombinations, although there probably was once. 

See if you can work out what is being tested in each class and name them better. I'm happy to pitch in with ideas.

Reply:
I've had a think and come up with the following refactor for the two test classes associated with the AbbreviatedNamesFunctions class.

One test class is concerned with splitting names around abbreviations which I've renamed to:
`AbbreviatedNamesFunctionsSplitAroundAbbreviatedNamesTest`
And the other is concerned with testing if a name input contains any abbreviated names:
`AbbreviatedNamesFunctionsDoesNotContainAbbreviatedNamesTest`

Let me know what you think.

Reply:
Yeah that's exactly what I had in mind. Both test the AbbreviatedNamesFunctions class. One tests the splitAroundAbbreviatedNames function and the other tests the doesNotContainAbbreviatedNames function so your names make sense.

Reply:
Resolved. Changes have been committed to this branch ready for another review.